holo-bgp/src/packet/message.rs

```diff
@@ -260,6 +262,8 @@ pub struct EncodeCxt {
 }
 
 // BGP message decoding context.
+#[derive(Debug)]
+#[derive(Arbitrary)]
```

Comment:
unnecessary derive

Reply:
`DecodeCxt::arbitrary()` is needed for multiple of the fuzzers to work.